src/TelegramLog.php

```diff
@@ -147,11 +147,10 @@ public static function initDebugLog($path)
     public static function getDebugLogTempStream()
     {
         if (self::$debug_log_temp_stream_handle === null) {
-            if (self::isDebugLogActive()) {
-                self::$debug_log_temp_stream_handle = fopen('php://temp', 'w+');
-            } else {
+            if (!self::isDebugLogActive()) {
```

Comment:
Since it's just a return, is there a need of "{}"?
It could be wrote on one line, no?

I bag your pardon for this message, but I've read a lot of your code last days, and I really like it, it's really nice, clean and i'd even say beautiful, so I'm just asking. )